system/Debug/Toolbar.php

```diff
@@ -310,6 +310,10 @@ protected function structureTimelineData(array $elements): array
      */
     protected function collectVarData(): array
     {
+        if (! $this->config->collectVarData) {
```

Comment:
So that it's not a BC break, we have to assume the value doesn't exist in the config file and treat it as `true` in that case. 

```suggestion
        if (! ($this->config->collectVarData ?? true)) {
```